tests/test/subtype.rs

```diff
@@ -544,10 +472,10 @@ fn generalize_slice() {
             }
         } yields {
             // Result should be identical to generalize_covariant_struct result.
-            "Unique; for<?U1> { substitution [?0 := [(&'^0.0 Uint(U32))]], lifetime constraints [\
+            expect![[r#"Unique; for<?U1> { substitution [?0 := [(&'^0.0 Uint(U32))]], lifetime constraints [InEnvironment { environment: Env([]), goal: '^0.0: '!1_0 }, InEnvironment { environment: Env([]), goal: '^0.0: '!1_1 }] }"#]], lifetime constraints [\
```

Comment:
Blessing is broken for this test case, since `expect-test` uses a regex search and stops at the `]]` at the end of "substitutions". `expect-test` should do better in these cases.